docs/api.md

```diff
@@ -1315,7 +1315,8 @@ Shortcut for [page.mainFrame().executionContext().queryObjects(prototypeHandle)]
     - `width` <[number]> width of clipping area
     - `height` <[number]> height of clipping area
   - `omitBackground` <[boolean]> Hides default white background and allows capturing screenshots with transparency. Defaults to `false`.
-- returns: <[Promise]<[Buffer]>> Promise which resolves to buffer with captured screenshot
+  - `encoding` <[string]> The encoding of the image, can be either `base64` or `binary`. Defaults to `binary`.
```

Comment:
In context of images and screenshots, `encoding` usually means JPG vs PNG, so I'd rather name name this  `base64` and make it boolean.

Reply:
@aslushnikov

1. I usually try to avoid boolean values when describing formats to support future compatibility with new values. 
2. `encoding` I believe that the correct term, even in image context, is `compression`, where `encoding` describes how the compression mechanism works. In this particular use case `encoding` with a `base64` value is the same property/value selection used in NodeJS for Buffers and Streams and describes how it works.